comp/autoscaling/datadogclient/impl/client.go

```diff
@@ -14,17 +14,22 @@ import (
 	configComponent "github.com/DataDog/datadog-agent/comp/core/config"
 	logComp "github.com/DataDog/datadog-agent/comp/core/log/def"
 	"github.com/DataDog/datadog-agent/comp/core/status"
+	"go.uber.org/fx"
```

Comment:
Going to drop this file from this PR, just testing because of an issue with testing :/